admin/database/database.go

```diff
@@ -224,6 +224,10 @@ type DB interface {
 	CountBillingProjectsForOrganization(ctx context.Context, orgID string, createdBefore time.Time) (int, error)
 	FindBillingUsageReportedOn(ctx context.Context) (time.Time, error)
 	UpdateBillingUsageReportedOn(ctx context.Context, usageReportedOn time.Time) error
+
+	FindBillingRepairedOn(ctx context.Context) (time.Time, error)
+	UpdateBillingRepairedOn(ctx context.Context, repairedOn time.Time) error
+	FindOrganizationIDsWithoutPaymentCreatedOnOrAfter(ctx context.Context, createdAfter time.Time) ([]string, error)
```

Comment:
Is there a reason that an org won't have a payment provider ID? If not, we could remove the `createdAfter` part here and just do `FindOrganizationIDsWithoutPaymentCustomerID`

Reply:
an org won't have payment provider id when there was any failure before updating the database with it or we failed in creating customer in stripe (may be some network issue or stripe server issue) etc. as stripe, orb customer creation and org updation with ids cannot be done in a transaction. If any or all step fails, repair job takes care of all scenarios. `createdAfter` is just an optimization to not look at orgs that were created before start time of last repair job.

Reply:
changed this to `FindOrganizationWithoutPaymentCustomerID` to get all orgs without valid payment customer id